### PR TITLE
create the index.html styles.css and images files

### PR DESCRIPTION
**Pull request by soesandarwin2201**
//portfolio setup
//create linters to the project
//create index.html
//create styles.css
//create images file
//and commit it

### CODE REVIEW

**Review by karlavdelgadof — CHANGES_REQUESTED** [review]
# STATUS: INVALID CODE REVIEW :x: :stop_sign:  

Hi there @soesandarwin2201  :wave: , 

While you made a great effort in this project :100: , unfortunately, I cannot proceed to review your code. 

## Invalid Code Review Request :warning:

You have submitted a project, that has linter errors, kindly run the Stylelint linter on your local terminal and quickly fix them so you can get a review! :eyes:

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Make it work and then make it better!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+    <!--home page-->
+    <section class="home">
+
+      <article class="home-content">
+      <h1 class="home-title">I'm Alexandra <br> Glad to see you!</h1>
+
+      <p class="home-main">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+
+      <span>LET'S CONNECT</span>
+
```
Comment (roshan-bajgain) [inline index.html:50]: - [ ] I would suggest you please use `hx` tag for the sub-title for better semantic HTML

File: index.html [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+        <ul class="navlink">
+          <li><a href="#index.html">Portfolio</a></li>
+          <li><a href="#about.html">About</a></li>
+          <li><a href="#contact.html">Contact</a></li>
+        </ul>
+      </nav>
+      <!--end of navbar-->
+
+      <div class="toggle">
+        <i class="uil uil-bars"></i>
```
Comment (roshan-bajgain) [inline index.html:36]: - [ ] I would suggest you please use the same color and font defined in the template.
### Your design
hisssssss
### Figma design
figma

File: index.html [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+    </header>
+
+    <!--home page-->
+    <section class="home">
+
+      <article class="home-content">
+      <h1 class="home-title">I'm Alexandra <br> Glad to see you!</h1>
+
+      <p class="home-main">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+
```
Comment (roshan-bajgain) [inline index.html:48]: - [ ] I would suggest you please the same color and the same font defined in the template. Please use `color: #344563` and `font-family: 'Poppins';`

File: index.html [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+      <span>LET'S CONNECT</span>
+
+      <ul class="social_links">
+        <li><a href=""><i class="uil uil-twitter"></i></a></li>
+        <li><a href=""><i class="uil uil-linkedin-alt"></i></a></li>
+        <li><a href=""><i class="uil uil-medium-m"></i></a></li>
+        <li><a href=""><i class="uil uil-github-alt"></i></a></li>
+        <li><a href=""><i class="fa fa-angellist" aria-hidden="true"></i></a></li>
+      </ul>
```
Comment (roshan-bajgain) [inline index.html:57]: - [ ] I would suggest you please use the same social media icons. You can use it by downloading it one by one from Figma and let's use the same font and same color for the `let's connect` subtitle.
### Your design
his
### Figma design
figma

File: index.html [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+            </div>
+            <button class="btn">See Project</button>
+          </div>
+          <img src="./images/Snapshoot Portfolio2.svg" alt="project" class="portfoilio-img">
+        </article> -->
+        <!--end of single article-->
+      <!-- </div>
+      
+    </section> -->
+  </div>
```
Comment (roshan-bajgain) [inline index.html:139]: - [ ] I would suggest you please remove the commented code as it make the code unclean

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @soesandarwin2201, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️

## Required Changes ♻️
- [ ] I would suggest you, please write the descriptive README file. Please use this [Template](https://github.com/microverseinc/readme-template/blob/master/README.md) but customize it
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**